docs/basic_training/containers.md

```diff
@@ -670,7 +670,7 @@ Nextflow automatically sets up an environment for the given package names listed
 
 !!! question "Exercise"
 
-    The tools `fastqc` and `salmon` are both available in BioContainers. Add the appropriate `container` directives to the `FASTQC` and `QUANTIFICATION` processes in `script5.nf` to use BioContainers instead of the container image you have been using in this training.
+    The tools `fastqc` and `salmon` are both available in BioContainers (`quay.io/biocontainers/fastqc:v0.11.5` and `quay.io/biocontainers/salmon:1.7.0--h84f40af_0`, respectively). Add the appropriate `container` directives to the `FASTQC` and `QUANTIFICATION` processes in `script5.nf` to use BioContainers instead of the container image you have been using in this training.
```

Comment:
I really like the idea of introducing them to seqera containers here instead of just giving them a bioconda path. Maybe something like this:
```suggestion
 [Seqera Containers](https://seqera.io/containers/) allows you to easily build a docker image for any tool available in conda or pip. 
1. Use Seqera Containers to build a docker container for each of:
 - fastqc v0.11.5
 - salmon v1.7.0
2. Add the appropriate `container` directives to the `FASTQC` and `QUANTIFICATION` processes in `script5.nf` to use BioContainers instead of the container image you have been using in this training.
```

Reply:
```suggestion
    The tools `fastqc` and `salmon` are both available in Seqera Containers (`community.wave.seqera.io/library/fastqc:0.12.1--5cfd0f3cb6760c42` and `community.wave.seqera.io/library/salmon:1.10.3--482593b6cd04c9b7`, respectively). Add the appropriate `container` directives to the `FASTQC` and `QUANTIFICATION` processes in `script5.nf` to use Seqera Containers instead of the container image you have been using in this training.
```